packages/web/src/common/store/upload/sagas.ts

```diff
@@ -1181,34 +1184,44 @@ export function* updateTrackAudioAsync(
   }
 
   const libs = yield* call(audiusBackendInstance.getAudiusLibsTyped)
-  const metadata = newTrackMetadata(
-    toUploadTrackMetadata({
-      ...track,
-      // @ts-ignore - stem_of should accept null instead of undefined
-      stem_of: track.stem_of ?? null
-    })
-  )
+  const baseMetadata = newTrackMetadata(toUploadTrackMetadata({ ...track }))
+  // Full Metadata with the updated from the edit form
```

Comment:
```suggestion
  // Full Metadata with the updates from the edit form
```

Reply:
will update in the next one to not run Ci again